Error when react-theming tries to import Hue type

Thanks for these great looking components. I'm getting an error from `react-theming` when building my project, however.

## Reality

A project using `@zendeskgarden/react-theming` will not build with `tsc` (TS 3.8.3). Instead, an error message about importing from `styled-components` occurs:

```
node_modules/@zendeskgarden/react-theming/dist/typings/utils/getColor.d.ts:7:24 - error TS2614: Module '"../../../../../@types/styled-components/ts3.7"' has no exported member 'Hue'. Did you mean to use 'import Hue from "../../../../../@types/styled-components/ts3.7"' instead?

7 import { DefaultTheme, Hue } from 'styled-components';
```

## Steps to Reproduce

There is [this minimal demonstration repo](https://github.com/toddoglethorpe/zengarden-issue).

But the short story is:

add
- `typescript`
- `@zendeskgarden/react-theming`
- `styled-components`
- `@types/styled-components` 

to a project. Import `ThemeProvider` from `react-theming` and try to compile (run `tsc`). Error occurs with `styled-components` v4 and v5.


<!--
  BONUS POINTS: spin up and share an example via
  https://codesandbox.io/s/github/zendeskgarden/react-components/tree/master/examples/codesandbox or https://codesandbox.io/s/new or https://stackblitz.com/fork/react.
-->

### Fine Print

- Component: react-theming
- Version: 8.2.0
- Browsers: not there yet

Thanks!

--- a/packages/theming/src/utils/getColor.ts
+++ b/packages/theming/src/utils/getColor.ts
@@ -9,7 +9,9 @@
 
 import DEFAULT_THEME from '../elements/theme';
 import { darken, lighten, rgba } from 'polished';
-import { DefaultTheme, Hue } from 'styled-components';
+import { DefaultTheme } from 'styled-components';
+
+type Hue = Record<number | string, string> | string;
 
 const DEFAULT_SHADE = 600;
 

--- a/utils/build/styled.d.ts
+++ b/utils/build/styled.d.ts
@@ -8,7 +8,7 @@
 import 'styled-components';
 
 declare module 'styled-components' {
-  export type Hue = Record<number | string, string> | string;
+  type Hue = Record<number | string, string> | string;
 
   /* eslint-disable-next-line @typescript-eslint/interface-name-prefix */
   export interface DefaultTheme {

